source/_components/device_tracker.netgear.markdown

```diff
@@ -22,17 +22,33 @@ To use this device tracker in your installation, add the following to your `conf
 # Example configuration.yaml entry
 device_tracker:
   - platform: netgear
-    host: YOUR_ROUTER_IP
+    url: http://YOUR_ROUTER_IP:5000
     username: YOUR_ADMIN_USERNAME
     password: YOUR_ADMIN_PASSWORD
+    devices:
+      - MY-LAPTOP
+      - 00:1A:2B:3C:44:55
+    exclude:
+      - MY-SERVER
+    accesspoints:
+      - 3C:44:55:00:1A:2B
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.